api/src/lists/guards/lists.guard.ts

```diff
@@ -10,7 +10,8 @@ export class ListAuthGuard implements CanActivate {
     const request = context.switchToHttp().getRequest();
     const { user } = request;
 
-    const listId = request.params.id || request.body.listId;
+    const listId =
+      request.params.id || request.body.listId || request.params.listId;
```

Comment:
This is ok for now, but at some point we'll have to unify this and make sure that a list's id only comes from `id` in params, not in the body or in params as `listId`. I guess this is tech debt.

Reply:
kk, I'll make an issue for us to go over all fields and rename to `id` as applicable.